app/controlplane/internal/service/auth.go

```diff
@@ -258,33 +274,42 @@ func (c *upstreamOIDCclaims) preferredEmail() string {
 
 func callbackHandler(svc *AuthService, w http.ResponseWriter, r *http.Request) *oauthResp {
 	ctx := context.Background()
-	// if OIDC provider returns an error, return it to and show it to the user
-	if desc := r.URL.Query().Get("error_description"); desc != "" {
-		return &oauthResp{http.StatusUnauthorized, errors.New(desc), true}
+	// if OIDC provider returns an error, redirect to the login page to and show it to the user
+	if desc := r.URL.Query().Get(oidcErrorParam); desc != "" {
+		redirectURL, err := url.Parse(svc.AuthURLs.Login)
```

Comment:
I am not sure we should redirect always but only if the login landing page is set no? https://github.com/chainloop-dev/chainloop/blob/e07cc0fd7674d9224fa5861007237285727746ac/app/controlplane/internal/conf/controlplane/config/v1/conf.proto#L193

Otherwise this will cause a redirect loop?